pkgs/development/libraries/science/math/osqp/default.nix

```
@@ -17,6 +17,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
+  patches = [ ./fix-cmake-header-path.diff ];
```

Could you consider opening a pull request upstream and adding comment with a link? You can refer to https://github.com/jtojnar/cmake-snips#creating-custom-install-path-options and  https://github.com/jtojnar/cmake-snips#assuming-cmake_install_dir-is-relative-path for explanations.

---

@jtojnar is that fine with you? https://github.com/NixOS/nixpkgs/pull/235850#issuecomment-1575276271 if so I would go ahead and merge this.

---

Ideally, this would still have a comment with link:

```suggestion
  patches = [
    # Fix header path including prefix twice.
    # https://github.com/osqp/osqp/pull/481
    ./fix-cmake-header-path.diff
  ];
```

---

@notalltim can you add the suggestion so we can get this merged?

---

I took the liberty of adding this so we can merge this.